### PR TITLE
Inject AWS_PROFILE if the Credential Provider is a Profile credential

### PR DESCRIPTION
**Pull request by kiiadi**
Screen Shot 2021-03-05 at 1 26 41 PM

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/core/terminal/OpenAwsLocalTerminal.kt [hunk 1]
```diff
@@ -41,7 +45,13 @@ class OpenAwsLocalTerminal : DumbAwareAction(message("aws.terminal.action"), mes
                         return@executeOnPooledThread
                     }
                     runInEdt {
-                        val runner = AwsLocalTerminalRunner(project, connection.region, credentials)
+                        val runner = AwsLocalTerminalRunner(project) { envs ->
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/core/terminal/OpenAwsLocalTerminal.kt:48]: Do we not have tests for this action?

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/terminal/OpenAwsLocalTerminal.kt:48]: :( no - we probably should.

*sigh* will add.

Comment (kiiadi) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/core/terminal/OpenAwsLocalTerminal.kt:48]: (well actually that's not true - we have a UI test)

**Review by hunterwerlla — APPROVED** [review]

**Pull request merged**